setup_pulp.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
```

Comment:
This file should probably be under .github, not at the root of the repo- maybe `.github/test-scripts` or something?

Reply:
Done